axelrod/tests/unit/test_cooperator.py

```diff
@@ -43,13 +39,11 @@ class TestTrickyCooperator(TestPlayer):
     }
 
     def test_strategy(self):
-        """Starts by cooperating."""
+        # Starts by cooperating.
```

Comment:
Sure but the docstring is either generic for all instances of `test_strategy`, i.e. `"""Tests that the strategy returns the expected actions."""` or it's essentially a repeat of the strategy's docstring. Is there a preference?